validator/client/runner.go

```diff
@@ -89,6 +121,12 @@ func run(ctx context.Context, v Validator) {
 			log.Info("Context canceled, stopping validator")
 			span.End()
 			return // Exit if context is canceled.
+		case blocksError := <-connectionErrorChannel:
+			if blocksError != nil && strings.Contains(blocksError.Error(), connectionIssueMsg) {
+				go v.ReceiveBlocks(ctx, connectionErrorChannel)
+				continue
+			}
+			log.WithError(blocksError).Fatal("block stream interrupted")
```

Comment:
should this be fatal ? @terencechain you introduced the block receiver in #8197, should the error from there be fatal or
can we continue running the validator client fine.

Reply:
cc @terencechain

Reply:
Continue running is fine. The worst case is it'll wait 4s to attest